<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignalDialog</class>
 <widget class="QDialog" name="SignalDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>775</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DAB Signal</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <widget class="QCustomPlot" name="spectrumPlot" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>100</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QCustomPlot" name="snrPlot" native="true"/>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="leftMargin">
      <number>12</number>
     </property>
     <property name="rightMargin">
      <number>12</number>
     </property>
     <property name="bottomMargin">
      <number>12</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="syncLabel">
         <property name="text">
          <string notr="true">X</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="snrLabel">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>SNR:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="snrValue">
         <property name="text">
          <string>12.4 dB</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="rfLevelVLine">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="rfLevelLaoyout">
       <item>
        <widget class="QLabel" name="rfLevelLabel">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>RF level:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="rfLevelValue">
         <property name="text">
          <string notr="true">-60.2 dBm</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="gainVLine">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="gainLayout">
       <item>
        <widget class="QLabel" name="gainLabel">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string notr="true">Gain:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="gainValue">
         <property name="text">
          <string notr="true">50 dB</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="frequencyLayout">
       <item>
        <widget class="QLabel" name="freqLabel">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Frequency:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="freqValue">
         <property name="text">
          <string notr="true">227360 kHz</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="freqOffsetLayout">
       <item>
        <widget class="QLabel" name="freqOffsetLabel">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Offset:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="freqOffsetValue">
         <property name="text">
          <string notr="true">-12346.7 Hz</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="ClickableLabel" name="menuLabel">
       <property name="toolTip">
        <string notr="true"/>
       </property>
       <property name="text">
        <string notr="true">M</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header>clickablelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
